app/src/main/java/com/github/multimatum_team/multimatum/MainActivity.kt

```diff
@@ -91,12 +98,12 @@ class MainActivity : AppCompatActivity() {
     This button trigger a basics notification in 1 sec
     here we use an id based on current time. We may use some parsed part of the corresponding deadline later.
     */
-    fun triggerNotification(view:View) {
-        var id = System.currentTimeMillis().toInt()
-        setNotification(System.currentTimeMillis()+4000, "asdf", "ouafouaf", id)
+    fun triggerNotification(view: View) {
+        val id = System.currentTimeMillis().toInt()
+        setNotification(System.currentTimeMillis() + 4000, "asdf", "ouafouaf", id)
```

Comment:
4000 should be a constant

Reply:
As before,
This is not my part, these were modified because of the auto-reformatting of android Studio